### ISSUE TITLE
Detection Output overlapp with surrounding words 

### ISSUE THREAD

**Issue by bely66** [open-mmlab/mmocr#1094]
Hi dears,
I have an issue in detection, the detection bbox overlaps with the word after It, and that happens most of the time with short words.
There're two areas I'm suspecting:
1. Training the detection network on more short words (I think it's unlikely the case because it's present in the dataset)
2. Adjust the padding when I extract crops from the original Image using the bbox, using this function [crop.py](https://github.com/open-mmlab/mmocr/blob/4f36bcd1aa5678eae7a44ef2407697e1d14ac176/mmocr/datasets/pipelines/crop.py#L87)

I don't know if there're other issues that needs to be looked at.

**Comment by gaotongxiao** [comment]
I guess the words are dense in your image. In this case, you might need a better training set that fits your domain since we pretrained the models on the datasets where the words are scattered.
